assets/src/domain/eventEditor/interfaces/types.ts

```diff
@@ -25,6 +25,12 @@ export interface EEJSData {
 	data: JsDataProps;
 }
 
+export enum DisplayDates {
```

Comment:
Maybe it should be in a `shared` folder :dunno:

Reply:
Moved: https://github.com/eventespresso/event-espresso-core/pull/2512/commits/ab74a0c727400a01cfbcb8edda4f9e53194dfa36